fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java

```diff
@@ -126,29 +153,37 @@ public boolean isSetIfNotExists() {
 
     public enum IndexType {
         BITMAP,
+        INVERTED,
+    }
 
+    public boolean isInvertedIndex() {
+        return (this.indexType == IndexType.INVERTED);
     }
 
     public void checkColumn(Column column, KeysType keysType) throws AnalysisException {
-        if (indexType == IndexType.BITMAP) {
+        if (indexType == IndexType.BITMAP || indexType == IndexType.INVERTED) {
```

Comment:
I think we can also add `BLOOM_FILTER` check this time, although it is not implemented yet.

Reply:
added